jina/types/arrays/document.py

```diff
@@ -299,6 +302,113 @@ def overriden_key(proto):
         else:
             self._pb_body.sort(*args, **kwargs)
 
+    def compute_distances(
+        self, X: 'np.ndarray', Y: 'np.ndarray', metric: str
+    ) -> 'np.ndarray':
```

Comment:
looks like a private & out-class method

Reply:
good point, I will leave it ouside the class